package.json

```diff
@@ -9,12 +9,24 @@
   "web-types": "dist/web-types.json",
   "type": "module",
   "types": "dist/shoelace.d.ts",
+  "jsdelivr": "./cdn/shoelace-autoloader.js",
+  "sideEffects": [
+    "./dist/shoelace.js",
+    "./dist/shoelace-autoloader.js",
+    "./dist/components/**/*.js",
```

Comment:
Hi, thanks for your work, but the sideEffects settings here seems to break the cherry picking usage such as:

```js
import '@shoelace-style/shoelace/dist/components/button/button.js';
```

Since the `dist/components/button/button.js` also imports `dist/chunks/*.js` which are not marked as `sideEffects`, which will be tree-shaked in production mode.

Reply:
Do you have a repo by chance? 

What bundler are you using? I tested mainly with Vite, Webpack (via CRA), and Parcel.

In Stackblitz using NPM + Vite, it seems to work.

https://stackblitz.com/edit/vitejs-vite-42iegf?file=main.js,package.json,index.html&terminal=dev

Reply:
I'm using Webpack. But your demo is just ok to reproduce it. Try `npm run build && npm run preview`. Tree-shaking only happens in production mode by default, so `npm run dev` is not the case.

Reply:
Also the docs for 2.6.0 version suggest to import for bundler like so `import '@shoelace-style/shoelace/dist/components/button/button.component.js';` but by doing this you also break the dev environment.

Using the same demo @KonnorRogers provided I've just changed the import for the button.

https://stackblitz.com/edit/vitejs-vite-n3cvck?file=main.js

Not sure if this is related to the problem or it's a docs error. @claviska

Reply:
@Antony92 🤔 where in the docs does it say to use the `.component.js` import? There's a note about auto-registering, but thats a different.